source/deployment/bulk-loading-data-format.rst

```diff
@@ -79,8 +79,8 @@ For clarity, the object is shown using regular JSON formatting, but in the data
 .. code-block:: javascript
```

Comment:
Above this code block, there is one more that doesn't have the proper quotes, below `
The following fragment is from a file that imports two teams, each with two channels, many users, and many posts.`

Reply:
Thanks, have fixed :)